include/miral/miral/minimal_window_manager.h

```diff
@@ -82,6 +82,10 @@ class MinimalWindowManager : public WindowManagementPolicy
 
     void advise_delete_app(miral::ApplicationInfo const& app_info) override;
 
+    void advise_new_window(WindowInfo const& app_info) override;
+
+    void advise_delete_window(WindowInfo const& app_info) override;
+
```

Comment:
We should document when these are introduced. 
```suggestion
    /// \remark Since MirAL 5.0
    void advise_new_window(WindowInfo const& app_info) override;

    /// \remark Since MirAL 5.0
    void advise_delete_window(WindowInfo const& app_info) override;

```